### PR TITLE
[fix] RL scheduler

### PR DESCRIPTION
**Pull request by chongruo**
fix an error in rl_scheduler #421

### CODE REVIEW

File: autogluon/scheduler/rl_scheduler.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ class RLScheduler(FIFOScheduler):
     >>> scheduler.join_jobs()
     >>> scheduler.get_training_curves(plot=True)
     """
-    def __init__(self, train_fn, args=None, resource=None, checkpoint='./exp/checkpoint.ag',
+    def __init__(self, train_fn, args=None, resource=None, searcher=None, checkpoint='./exp/checkpoint.ag',
```
Comment (zhanghang1989) [inline autogluon/scheduler/rl_scheduler.py:73]: can you also add:

```
if not isinstance(searcher, RLSearcher):
```
before this line https://github.com/awslabs/autogluon/blob/2ffc14ab69aa1d3224a36347b0373ee062684173/autogluon/scheduler/rl_scheduler.py#L84

Comment (chongruo) [reply autogluon/scheduler/rl_scheduler.py:73]: have added. Thanks

**Comment by szha** [comment]
Job PR-427-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-427/1/index.html

**Comment by szha** [comment]
Job PR-427-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-427/2/index.html

**Pull request merged**